src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRISE = 40;
+  const SMALLSALE = 20;
+  const BIGSALE = 50;
```

Comment:
In JS we don't use uppercase for all constants. We use uppercase for some specific constants, for example:
`const PI = 3.14...`
`const BASE_URL = https://github.com/`
etc.